core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/docker.yml

```diff
@@ -30,9 +30,9 @@
   when:
     - is_docker_updatable
 
-# Remove Docker in version < 18.09 on RedHat before installing v18.09 to avoid package conflict error like below
+# Remove Docker in version < 20.10 on RedHat before installing v20.10 to avoid package conflict error like below
 # 'file /usr/bin/docker from install of docker-ce-cli-1:18.09.9-3.el7.x86_64 conflicts with file from package docker-ce-17.03.2.ce-1.el7.centos.x86_64'
-- name: Remove Docker in version < 18.09 (RedHat)
+- name: Remove Docker in version < 20.10 (RedHat)
```

Comment:
We may no need this task anymore. It was added because `docker-ce` was split into two packages starting with version 18.09 what caused conflict. In the new version I see new package `docker-ce-rootless-extras` which also may cause conflict (if they moved some files to it). Let's test without this block.

Reply:
In future each time when such thing happens, we may need constantly add/remove this task. WDYT about leaving that for simplier upgrade?